Mathlib/Tactic/FieldSimp.lean

```diff
@@ -135,7 +135,7 @@ elab_rules : tactic
 | `(tactic| field_simp $[$cfg:config]? $[$dis:discharger]? $[only%$only?]?
     $[$sa:simpArgs]? $[$loc:location]?) => do
   let cfg ← elabSimpConfig (cfg.getD ⟨.missing⟩) .simp
-  let loc := expandOptLocation (loc.getD ⟨.missing⟩)
+  let loc := expandOptLocation (mkOptionalNode loc)
```

Comment:
should something similar happen down on line 154, where we currently have `sa.getD ⟨.missing⟩`?

Reply:
That seems to work ok as is to me, I can pass additional simp lemmas to field simp and it does use them.